unity-renderer/Assets/Scripts/MainScripts/DCL/Analytics/SocialAnalytics/SocialAnalytics.cs

```diff
@@ -0,0 +1,270 @@
+            data.Add("from", fromPlayerType.ToString());
+            data.Add("to", toPlayerType.ToString());
+            data.Add("source", source.ToString());
+
+            GenericAnalytics.SendAnalytic(FRIEND_DELETED, data);
+        }
+
+        public void SendPassportOpen()
+        {
+            GenericAnalytics.SendAnalytic(PASSPORT_OPENED);
+        }
+
+        public void SendPassportClose(double timeSpent)
+        {
+            Dictionary<string, string> data = new Dictionary<string, string>();
+            data.Add("time_spent", timeSpent.ToString());
+
+            GenericAnalytics.SendAnalytic(PASSPORT_CLOSED, data);
+        }
+
+        public void SendPlayerBlocked(bool isFriend, FriendActionSource source)
+        {
+            Dictionary<string, string> data = new Dictionary<string, string>();
+            data.Add("friend", isFriend.ToString());
+            data.Add("source", source.ToString());
+
+            GenericAnalytics.SendAnalytic(PLAYER_BLOCKED, data);
+        }
+
+        public void SendPlayerUnblocked(bool isFriend, FriendActionSource source)
+        {
+            Dictionary<string, string> data = new Dictionary<string, string>();
+            data.Add("friend", isFriend.ToString());
+            data.Add("source", source.ToString());
+
+            GenericAnalytics.SendAnalytic(PLAYER_UNBLOCKED, data);
+        }
+
+        public void SendPlayerReport(PlayerReportIssueType issueType, double messageLength, FriendActionSource source)
+        {
+            Dictionary<string, string> data = new Dictionary<string, string>();
+            data.Add("issue_type", issueType.ToString());
+            data.Add("text_length", messageLength.ToString());
+            data.Add("source", source.ToString());
+
+            GenericAnalytics.SendAnalytic(PLAYER_REPORT, data);
+        }
+
+        public void SendPlayerJoin(FriendActionSource source)
+        {
+            Dictionary<string, string> data = new Dictionary<string, string>();
+            data.Add("source", source.ToString());
+
+            GenericAnalytics.SendAnalytic(PLAYER_JOIN, data);
+        }
+
+        public void SendPlayEmote(string emoteId, string emoteName, string rarity, bool isBaseEmote, UserProfile.EmoteSource source, string parcelLocation)
+        {
+            Dictionary<string, string> data = new Dictionary<string, string>();
+            data.Add("id", emoteId);
+            data.Add("name", emoteName);
+            data.Add("rarity", rarity);
+            data.Add("isBase", isBaseEmote.ToString());
+            data.Add("source", source.ToString());
+            data.Add("parcel_location", parcelLocation);
+
+            GenericAnalytics.SendAnalytic(PLAY_EMOTE, data);
+        }
+
+        private PlayerType? GetPlayerTypeByUserId(string userId)
+        {
+            if (string.IsNullOrEmpty(userId))
+                return null;
+
+            UserProfile userProfile = UserProfileController.GetProfileByUserId(userId);
```

Comment:
please inject `IUserProfileBridge` and replace the call to `userProfileBridge.Get(userId)`

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2257/commits/f0000e7242a9b625c830c83258d83fc88e9baf1a